Funcky/Sequence/CycleRange.cs

```diff
@@ -14,6 +16,19 @@ public static partial class Sequence
         /// <returns>Returns an infinite IEnumerable repeating the same sequence of elements.</returns>
         public static IEnumerable<TItem> CycleRange<TItem>(IEnumerable<TItem> sequence)
```

Comment:
Document that `sequence` must have at least one element.

Reply:
Adressed in f7135559